fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_COCKPIT.xml

```diff
@@ -4982,6 +4982,9 @@
                 </Component>
 
                 <Component ID="Overhead_Reset_Panel">
+                <UseTemplate Name="FBW_Airbus_RESET_PANEL_BUTTON">
```

Comment:
We should probably extract this to its own `.xml` file at some point, like other newer behaviours. Probably not in this PR though